docs/natspec-format.rst

```diff
@@ -0,0 +1,202 @@
+
+This example shows a contract and a function using all available tags.
+Note: NatSpec currently does NOT apply to variables (see
+`solidity#3418 <https://github.com/ethereum/solidity/issues/3418>`__),
+even if they are declared public and therefore do affect ABI. Note:
+NatSpec currently only interprets tags functions if they are external or
+public. You are welcome to use similar comments for your internal and
+private functions, but those will not be parsed.
+
+.. code:: solidity
+
+   pragma solidity ^0.5.1;
+
+   /// @title A simulator for Bug Bunny, the most famous Rabbit
+   /// @author Warned Bros
+   /// @notice You can use this contract for only the most basic simulation
+   /// @dev All function calls are currently implement without side effects
+   contract BugsBunny {
+       /// @author Bob Clampett
+       /// @notice Determine if Bugs will accept `(food)` to eat
+       /// @dev String comparison may be inefficient
+       /// @param food The name of a food to evaluate (English)
+       /// @return true if Bugs will eat it, false otherwise
+       function doesEat(string calldata food) external pure returns (bool) {
+           return keccak256(bytes(food)) == keccak256("carrot");
+       }
+   }
+
+.. _header-n12:
+
+Tags
+====
+
+All tags are optional. The following table explains the purpose of each
+NatSpec tag and where it may be used.
+
+=========== =============================================================================== =============================
+Tag                                                                                         Context
+=========== =============================================================================== =============================
+``@title``  A title that should describe the contract/interface                             contract, interface
+``@author`` The name of the author                                                          contract, interface, function
+``@notice`` Explain to an end user what this does                                           contract, interface, function
+``@dev``    Explain to a developer any extra details                                        contract, interface, function
+``@param``  Documents a parameter just like in doxygen (must be followed by parameter name) function
+``@return`` Documents the return type of a contract's function                              function
+=========== =============================================================================== =============================
+
+If your function returns a tuple, like ``(int quotient, int remainder)``
+then use multiple ``@return`` statements in the same format as the
+``@param`` statements.
+
+.. _header-n44:
+
+Dynamic expressions
+-------------------
+
+In function documentation, you may use dynamic expressions for all tags.
+Example:
+
+.. code::
+
+   /// @notice Send `(valueInmGAV / 1000).fixed(0,3)` GAV from the account of
```

Comment:
It would be great if this could refer to the example above and not some weird coin nobody has heard of before ;)

Reply:
In addition to the example, things like `.fixed(...)` have never worked.